### ISSUE TITLE
URL with ?amp query var unexpectedly fails AMP URL test

### ISSUE THREAD

**Issue by westonruter** [ampproject/ampbench#113]
This URL is failing the AMP URL test: https://www.history101.com/vikingfacts/?amp

Results: https://ampbench.appspot.com/validate?url=https%3A%2F%2Fwww.history101.com%2Fvikingfacts%2F%3Famp

![image](https://user-images.githubusercontent.com/134745/57171347-62cd0180-6dc8-11e9-8476-81d79f37cd19.png)

Nevertheless, this AMP URL also with `?amp` is passing the AMP URL test:

https://make.xwp.co/2019/03/15/7-experiences-that-create-team/?amp

Results: https://ampbench.appspot.com/validate?url=https%3A%2F%2Fmake.xwp.co%2F2019%2F03%2F15%2F7-experiences-that-create-team%2F%3Famp

![image](https://user-images.githubusercontent.com/134745/57171362-82642a00-6dc8-11e9-8618-0e48a5494515.png)

What's the difference? This is of concern for the AMP WordPress plugin because we are moving to `?amp` for all paired AMP URLs.

**Comment by swissspidy** [comment]
I think the issue is that the "AMP URL" label is misleading. This is not validating the format of the URL, but actually checks whether that page is valid AMP.

The "FAIL" comes from the AMP validation output that you can see when scrolling down on https://ampbench.appspot.com/validate?url=https%3A%2F%2Fwww.history101.com%2Fvikingfacts%2F%3Famp:

<img width="1137" alt="Screenshot 2019-05-04 at 12 10 28" src="https://user-images.githubusercontent.com/841956/57177589-a45fba00-6e65-11e9-9374-c483f85adcde.png">

tl;dr everything works correctly in AMP Bench

**Comment by ithinkihaveacat** [comment]
@swissspidy is correct: the "AMP URL" status is misleading. It doesn't do anything to validating the URL itself  (AMP has no requirements here) but whether the document itself is valid. I guess it should read "AMP valid?" or similar. Will create a PR to fix this.

**Comment by westonruter** [comment]
@ithinkihaveacat Humm. The main validator doesn't report an error:

https://validator.ampproject.org/#url=https%3A%2F%2Fwww.history101.com%2Fvikingfacts%2F%3Famp

Is AMP Bench's validator stale?

**Comment by ithinkihaveacat** [comment]
Uh yes you're right, the validator was stale. Fixing _that_ problem is #97…
